util/fix/fix.go

```diff
@@ -0,0 +1,11 @@
+package fix
+
+import (
+	"os"
+	"strings"
+)
+
+// Path resolves "~" intro the real home path
+func Path(path string) string {
+	return strings.Replace(path, "~", os.Getenv("HOME"), 1)
```

Comment:
@ivanilves check the golang package ` os/user` you can get home directory there, maybe more elegant

Reply:
good observation!